src/utils/getLatestDate.ts

```diff
@@ -0,0 +1,33 @@
+const getCurrentDate = (): string => {
+    const date = new Date();
+    const month = [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December',
+    ];
+
+    const daysInWeek = [
+        'Sunday',
+        'Monday',
+        'Tuesday',
+        'Wednesday',
+        'Thursday',
+        'Friday',
+        'Saturday',
+    ];
```

Comment:
we shouldn't have this data in the function.

Reply:
done